packages/pg-vector-selector/src/AgenticaPgVectorSelector.ts

```diff
@@ -242,7 +242,7 @@ export namespace AgenticaPgVectorSelector {
         .then(ChatGptCompletionMessageUtil.merge);
 
       selectCompletion.choices
-        .filter(v => !!v.message.tool_calls)
+        .filter(v => v.message.tool_calls != null && v.message.tool_calls.length > 0)
```

Comment:
fix `strict-boolean-expressions` error as well

Reply:
we don't know we need `v.message.tool_calls.length > 0` but i don't want to change the whole meaning so I added.
We may remove it later, but we need test to prove it

Reply:
You don't have to.
If the length of the array was empty, forEach wouldn't work.

That's what the code meant.

Reply:
Okay! done with abafa1588eba